src/middlewares/checkUser.js

```diff
@@ -0,0 +1,35 @@
+import models from '../api/models';
+import status from '../helpers/constants/status.codes';
+
+const { User } = models;
+/**
+ * check if user exist
+ *
+ * @export
+ * @class CheckUser
+ */
+class CheckUser {
+  /**
+   * this is a middleware which checks if the user is registered in to our database.
+   *
+   * @static
+   * @param {object} req the request
+   * @param {object} res the response
+   * @param { object } next the next route controller to be called
+   * @memberof CheckUser
+   * @returns {Object} res
+   */
+  static async checkIfUserExist(req, res, next) {
```

Comment:
@Ramadhan0 also, I think a middleware already exists for this. Ask @MCFrank16

Reply:
The one @MCFrank16 created checks if the user is the ```Author```